src/Mpdf.php

```diff
@@ -1550,6 +1550,16 @@ public function cleanup()
 	{
 		mb_internal_encoding($this->originalMbEnc);
 		@mb_regex_encoding($this->originalMbRegexEnc);
+
+		// this will free up the readers, based on code from Setasign's FpdiTrait::cleanUp()
+		$allReaders = false;
+		$readers = $allReaders ? array_keys($this->readers) : $this->createdReaders;
```

Comment:
The allReaders variable is completely unnecessary, just use `array_keys($this->readers)` directly, please.

Also, please, squash all the commits and forcepush to the branch.

Reply:
Ok, sure. 

My thinking was since we or at least I'm not too sure what `$this->readers` or `createdReaders` really do, maintaining the entire snippet integrity can be helpful down the road.

I have not used squash or forcepush before, let me give it a try. Thank you for the patience.

Reply:
Ok, seems I managed both on first try. Since `$allReaders = false`, we should be looping over `$this->createdReaders`. Thanks.